easybuild/easyconfigs/a/ANTLR/ANTLR-2.7.7-foss-2016a-Python-2.7.11.eb

```diff
@@ -0,0 +1,34 @@
+easyblock = 'ConfigureMake'
+
+name = 'ANTLR'
+version = "2.7.7"
+
+homepage = 'http://www.antlr2.org/'
+description = """ANTLR, ANother Tool for Language Recognition, (formerly PCCTS)
+ is a language tool that provides a framework for constructing recognizers,
+ compilers, and translators from grammatical descriptions containing
+ Java, C#, C++, or Python actions."""
+
+toolchain = {'name': 'foss', 'version': '2016a'}
+
+source_urls = ['http://www.antlr2.org/download/']
+sources = [SOURCELOWER_TAR_GZ]
+
+patches = ['%(name)s-%(version)s_includes.patch']
+
+pythonversion = '2.7.11'
+versionsuffix = "-Python-%s" % pythonversion
```

Comment:
please use `%(pyver)s` (supported since EasyBuild v2.7.0):

```python
name = 'ANTLR'
version = '2.7.7'
versionsuffix = '-Python-%(pyver)s'
...
dependencies = [
    ('Python', '2.7.11'),
    ...
]
```